.github/workflows/manage.yml

```diff
@@ -0,0 +1,54 @@
+name: Manage workflows
```

Comment:
Why is this necessary?

Reply:
Our current PR process involves a developer committing multiple times after a PR is open. We do not want each of them to start up a new GitHub Actions. The functions of `manage.yml` are twofold: 1) when GitHub Actions is started, if it sees previously running (or queued) tests of the same same user and head branch, it will cancel them; 2) if a developer specify skip-ci in his/her commit message, GitHub Actions will not be started.